README.md

```diff
@@ -275,11 +275,17 @@ A curated list of cryptography resources and links.
 - [Google Keyczar](https://github.com/google/keyczar) - Easy to use, yet safe encryption framework with key versioning.
 - [Google Tink](https://github.com/google/tink) - A small crypto library that provides a safe, simple, agile and fast way to accomplish some common crypto tasks.
 - [Java Themis](https://github.com/cossacklabs/themis/wiki/Java-and-Android-Howto) - Java/Android wrapper on Themis. High level crypto library for storing data (AES), secure messaging (ECC + ECDSA / RSA + PSS + PKCS#7) and session-oriented, forward secrecy data exchange (ECDH key agreement, ECC & AES encryption).
+- [jbcrypt](https://github.com/djmdjm/jBCrypt) - jBCrypt is an implementation the OpenBSD Blowfish password hashing
```

Comment:
This library is not maintained, why do you think that it is worth adding?

Reply:
This library is alive. You can see maven 'last updated'. I cannot investigate what the github repo is correct. I have replaced with projects home page.